src/Template/Element/Flash/default.ctp

```diff
@@ -0,0 +1,13 @@
+<?php
+$class = array_unique((array)$params['class']);
+$message = h($message);
+
+if (in_array('alert-dismissible', $class)) {
+    $button = <<<BUTTON
+<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+BUTTON;
```

Comment:
Any reason for using raw HTML markup instead of a tag generation helper method (i.e. `Form::button()`)?